admin/css/views/status.css

```
@@ -841,7 +841,9 @@ body.dark-theme .connection-item--flippable:hover .connection-flip-face {
     align-items: center;
     gap: 16px;
     overflow: hidden;
-    margin-bottom: 16px;
+    /* 不追加底部外边距：跑马灯与下方卡片的间距统一交给
+       .dashboard-grid 的 gap（24px）接管*/
```

Stylelint (17.14.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)